<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>621</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>50</y>
      <width>151</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>   0   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line0"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>   1   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line1"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>   2   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>   3   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line3"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>   4   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line4"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>   5   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line5"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>   6   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line6"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>   7   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line7"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>   8   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line8"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="algoBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Hemming Algo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hebb Algo</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="study">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Study Nets</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
